README.md

```diff
@@ -5,37 +5,39 @@
 [Core ML Tools](https://coremltools.readme.io/docs)
 =======================
 
-Core ML is an Apple framework to integrate machine learning models into your
-app. Core ML provides a unified representation for all models. Your app uses
-Core ML APIs and user data to make predictions, and to fine-tune models, all on
-the user’s device. Core ML optimizes on-device performance by leveraging the
-CPU, GPU, and Neural Engine while minimizing its memory footprint and power
-consumption. Running a model strictly on the user’s device removes any need for
-a network connection, which helps keep the user’s data private and your app
-responsive.
-
-[Core ML tools](https://coremltools.readme.io/docs#what-is-coremltools) contains all supporting tools for [Core ML model
-conversion](https://coremltools.readme.io/docs), editing and validation. This includes deep learning frameworks like
-TensorFlow, PyTorch, Keras, Caffe as well as classical machine learning
-frameworks like LIBSVM, scikit-learn, and XGBoost.
+Use *coremltools* to convert machine learning models from third-party libraries to the Core ML format. The Python package contains the supporting tools for converting models from training libraries such as the following:
+
+* [TensorFlow 1](https://www.tensorflow.org/versions/r1.15/api_docs/python/tf) and [TensorFlow 2](https://www.tensorflow.org/api_docs)
```

Comment:
I would probably names these `TensorFlow 1.X` and `TensforFlow 2.X`. I would probably also make them two separate bullet points.

Reply:
OK done.